src/Dist_Archive_Command.php

```diff
@@ -195,6 +198,11 @@ public function __invoke( $args, $assoc_args ) {
 		}
 		$archive_absolute_filepath = "{$archive_path}/{$archive_filename}";
 
+		if ( file_exists( $archive_absolute_filepath ) ) {
+			WP_CLI::warning( "The file '{$archive_absolute_filepath}' already exists." );
+			WP_CLI::confirm( 'Do you want to overwrite it?', $assoc_args );
+		}
```

Comment:
I'm not quite sure `WP_CLI::confirm()` is right.

Here's the behavior for `wp scaffold package-readme`:

```
$ wp scaffold package-readme ./
Warning: File already exists
.//README.md
Skip this file, or replace it with scaffolding?[s/r]:
```

Here's the behavior for `wp core download`:

```
$ wp core download
Error: WordPress files seem to already be present here.
$ wp core download --force
Downloading WordPress 6.4.1 (en_US)...
md5 hash verified: 5f9044e6b3f78f1bbdf85fed0244f778
Success: WordPress downloaded.
```

@BrianHenryIE What prior art do you think we should follow?

Reply:
@danielbachhuber , I refactored here:
https://github.com/wp-cli/dist-archive-command/pull/85/commits/3ef862dcc59bce77e6e7c4f108de539f1d574065

And updated the PR description with the `Example command execution`.

Reply:
@sejas Nice job on the tests!

I think I like this output but I'm not sure I love it:

```
$  wp dist-archive ./
Warning: File already exists
/Users/danielbachhuber/wordpress-plugins/one-time-login.zip
Do you want to skip or replace it with a new archive? [s/r]:
```

@BrianHenryIE @wp-cli/committers Any opinions on how it could be improved?

Reply:
@danielbachhuber , what change would you make to love the output?

Reply:
I don't think I have a better suggestion than the current form, which seems fine

Reply:
I changed 'File already exists' to 'Archive file already exists': 58105cf

Now I'm happy 😁